audbackend/core/backend/base.py

```diff
@@ -71,6 +71,70 @@ def checksum(
             path,
         )
 
+    def _copy_file(
+            self,
+            src_path: str,
+            dst_path: str,
+            verbose: bool,
+    ):  # pragma: no Windows cover
+        r"""Copy file on backend.
+
+        A default implementation is provided,
+        which temporarily gets the file from the backend
+        and afterward puts it to the new location.
+        It is recommended to overwrite the function
+        if backend supports a native way to copy files.
+
+        """
+        with tempfile.TemporaryDirectory() as tmp:
+            tmp_path = audeer.path(tmp, '~')
+            tmp_path = self.get_file(src_path, tmp_path, verbose=verbose)
+            self.put_file(tmp_path, dst_path, verbose=verbose)
+
+    def copy_file(
+            self,
+            src_path: str,
+            dst_path: str,
+            *,
+            verbose: bool = False,
+    ):
+        r"""Copy file on backend.
+
+        If ``dst_path`` exists
+        with a different checksum,
+        it is overwritten,
+        or otherwise,
+        the operation is silently skipped.
```

Comment:
Maybe change to

```suggestion
        If ``dst_path`` exists
        and has a different checksum,
        it is overwritten.
        Otherwise,
        the operation is silently skipped.
```